WalletWasabi.Fluent/ViewModels/CoinControl/Core/ColumnFactory.cs

```diff
@@ -0,0 +1,111 @@
+using System.Linq;
+using Avalonia.Controls;
+using Avalonia.Controls.Models.TreeDataGrid;
+using Avalonia.Layout;
+using NBitcoin;
+using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Fluent.ViewModels.CoinControl.Core.Cells;
+using WalletWasabi.Fluent.ViewModels.CoinControl.Core.Headers;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl.Core;
+
+public static class ColumnFactory
```

Comment:
Why separate static class, wouldn't better to keep it in same file and TDG source is defined?

Reply:
This class previously was shared by 2 different ViewModels and avoided duplication. I moved the methods.